packages/@okta/vuepress-site/guides/sign-into-mobile-app/get-user-info/iOS/getinfo.md

```diff
@@ -11,4 +11,4 @@ stateManager?.getUser { response, error in
 }
 ```
 
-The user info (of type 'Dictionary') is returned as a closure parameter. Possible properties are listed [here](https://developer.okta.com/docs/api/resources/oidc/#response-example-success-5).
+The user info (of type `Dictionary`) is returned as a closure parameter.
```

Comment:
@IldarAbdullin-okta   Can you add an example of pulling the user's email and name out of this object so it can be shown in the UI?

Reply:
Sure, will add example